pybossa/forms/forms.py

```diff
@@ -99,6 +99,7 @@ class ProjectUpdateForm(ProjectForm):
                                 message=lazy_gettext(
                                     "You must provide a description.")),
                              validators.Length(max=255)])
+    short_name = TextField(label=None, widget=HiddenInput())    
```

Comment:
I think that if this is left here, `short_name` can still be changed via `/project/<short_name>/update`

Reply:
If that is the case then id can be changed too? line 96

Reply:
if that is the case then  id can be changed too? line 96

Reply:
No, sorry - I didn't see https://github.com/bloomberg/pybossa/pull/190/files#diff-8fa3bf883c1a925427ca6a3b4587f452L603

Reply:
cool!